src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -38,8 +38,7 @@ public class AddCommand extends Command {
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney "
-            + PREFIX_LOG + "20 Jan 2022|First visit";
+            + PREFIX_TAG + "owesMoney ";
```

Comment:
i think this for the tag example a bit weird to label a patient as owesmoney, can consider changing in next iteration